.sequelizerc

```diff
@@ -2,7 +2,5 @@ const path = require('path')
 
 module.exports = {
   "config": path.resolve('./server/config', 'database-config.js'),
-  "models-path": path.resolve('./server/models'),
-  "seeders-path": path.resolve('./server/seeders'),
-  "migrations-path": path.resolve('./server/migrations')
```

Comment:
I'm not sure whether to keep this file.  It enables the sequelize-cli, so for example if you want to create a new model with the cli, it generates the model file and the migration.  Or you can generate empty migrations and seeders.

We're not going to be creating any migrations anymore and I don't think you can create models without migrations.  So unless we add seeders it's not adding any value.

Reply:
good spot, I've removed the liquibase specific files and the references in the package.json.